Nodejs/Product/Nodejs/Commands/SendFeedbackCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿//*********************************************************//
+//    Copyright (c) Microsoft. All rights reserved.
+//    
+//    Apache 2.0 License
+//    
+//    You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//    
+//    Unless required by applicable law or agreed to in writing, software 
+//    distributed under the License is distributed on an "AS IS" BASIS, 
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
+//    implied. See the License for the specific language governing 
+//    permissions and limitations under the License.
+//
+//*********************************************************//
+
+using System;
+using Microsoft.VisualStudioTools;
+using System.Diagnostics;
+
+namespace Microsoft.NodejsTools.Commands {
+    internal sealed class SendFeedbackCommand : Command {
+        public override void DoCommand(object sender, EventArgs args) {
+            var uri = new Uri(@"https://aka.ms/ntvs-feedback");
```

Comment:
fwlink instead of aka.ms - aka.ms is only for vanity urls / social media, which isn't necessary for the product. The nice thing is that the aka.ms tool now lets you easily switch the domain create fwlinks (which was the other reason we might use it in-product :smiley:)